interfaces/inc/file.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef SYNC_H
+#define SYNC_H
+
+#ifdef __cplusplus
+#include <cstdbool>
+#else
+#include <stdbool.h>
+#endif
+
+#include "umock_c/umock_c_prod.h"
+#include "constbuffer.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define FILE_DATA_RESULT_VALUES \
+    FILE_DATA_FAILURE, \

```

Comment:
>FILE_DATA_FAILURE [](start = 4, length = 17)

What's the difference between FAILURE and INVALID?

Reply:
I just copied this from bsdl_file_ll.h but it seems like invalid is not used anywhere bsdl_file_ll.c. Should I just remove it?

---
In reply to: [445224083](https://github.com/Azure/azure-c-pal/pull/7#discussion_r445224083) [](ancestors = 445224083)